src/analysis/mesh/qgsmeshcalculator.h

```diff
@@ -155,13 +167,14 @@ class ANALYSIS_EXPORT QgsMeshCalculator
      */
     QgsMeshCalculator( const QString &formulaString,
                        const QString &outputGroupName,
-                       double startTime,
-                       double endTime,
                        const QgsRectangle &outputExtent,
-                       QgsMeshLayer *layer );
+                       const QgsMeshCalculator::Destination &destination,
```

Comment:
we should not change the API, best to keep the old API and create additional function with the new api. Old api can be marked deprecated for example.

Reply:
it is a change from the "memory" PR.

Reply:
api should be kept from 3.14 release. we can add new function(s). so you say this particular function was introduced with in-memory PR and not present in 3.14 release?

Reply:
that's it